src/components/landing_page/Event.js

```diff
@@ -34,18 +34,18 @@ function EventTab(props){
             <div>
                 <h1 className={classes.past1}>Fall 2020 Coffee Chat Program</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Coffee Chats are an opportunity for students to connect with multiple experienced upper year students to explore more areas of technology.
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
                 <h1 className={classes.past1}>Fall 2020 Mock Interviews</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Practice your interview skills with upper-year students who have scored co-ops at amazing companies, big and small!
+                    {/* <Button className={classes.past1_button}>Learn More</Button> */}
                 </p>
                 <h1 className={classes.past1}>Winter 2021 Waterloo Coop Intro</h1>
                 <p className={classes.past1_desc}>
-                    A description of the project 
-                    <Button className={classes.past1_button}>Learn More</Button>
+                    Introduction to Waterloo coop structure and tips in how to be prepared and apply for jobs. 
```

Comment:
```suggestion
                    Introduction to Waterloo co-op structure and tips in how to be prepared and apply for jobs. 
```
Should make sure all usages of "co-op" are consistent.